### PR TITLE
add images to forum posts

### PR DESCRIPTION
**Pull request by aman3014**
Adds images to the forum. Each post can now have one image associated to it. This image can be added either from the gallery or by taking a picture with the Camera. When seeing the post, the associated picture is displayed. If there isn't an associated picture, a default image (https://en.wikipedia.org/wiki/Roman_Forum#/media/File:Foro_Romano_Musei_Capitolini_Roma.jpg) is shown.

The low diff coverage is due to methods like compression, upload, and image display (using Picasso). But since these are API calls, they don't constitute that many LoCs which is why the total coverage doesn't change at all.

### CODE REVIEW

File: app/src/main/java/com/github/epfl/meili/forum/ForumActivity.kt [hunk 1]
```diff
@@ -63,28 +93,34 @@ class ForumActivity : AppCompatActivity() {
...
             else -> openPost(view.findViewById(R.id.post_id))
         }
     }
 
     private fun openPost(view: View) {
         val postId: String = (view as TextView).text.toString()
         val intent: Intent = Intent(this, PostActivity::class.java)
-                .putExtra("Post", viewModel.getElements().value?.get(postId))
+                .putExtra(Post.TAG, viewModel.getElements().value?.get(postId))
+                .putExtra("PostId", postId)
```
Comment (tberkane) [inline app/src/main/java/com/github/epfl/meili/forum/ForumActivity.kt:123]: Nitpick: create a constant for this.

File: app/src/main/java/com/github/epfl/meili/forum/ForumActivity.kt [hunk 2]
```diff
@@ -146,6 +186,37 @@ class ForumActivity : AppCompatActivity() {
...
+            }
+
+            runOnUiThread {
+                this.bitmap = bitmap
+                displayImageView.setImageBitmap(bitmap)
+            }
+        }
+    }
+
+    private fun compressed(bitmap: Bitmap): ByteArray {
```
Comment (tberkane) [inline app/src/main/java/com/github/epfl/meili/forum/ForumActivity.kt:204]: Nitpick: this method name could be a bit more descriptive

**Review by tberkane — APPROVED** [review]
Looks good!

**Review by Bwananah — APPROVED** [review]
Looks good

**Pull request merged**